### DISCUSSION TITLE
Demo Gif

### DISCUSSION THREAD

**Discussion by mrjones2014** [mrjones2014/smart-splits.nvim#5] [General]
![demo-min](https://user-images.githubusercontent.com/8648891/159472445-ef680c42-f0fc-4c21-9ab7-0590a89da95b.gif)

**Comment by mrjones2014** [comment]
Updated demo gif with tmux integration.

![CleanShot 2022-11-15 at 08 13 20](https://user-images.githubusercontent.com/8648891/201928611-4338e3cb-cca9-4e15-92c6-0405b7072279.gif)

**Comment by mrjones2014** [comment]
Demo gif of directionally swapping buffers:

![CleanShot 2023-02-21 at 09 21 32](https://user-images.githubusercontent.com/8648891/220370953-5d0a279a-cc7e-4642-9548-bf4de274b938.gif)

**Reply by A7R7** [reply]
I love your plugin! but how did you managed to change the bg and border color of the active window?

**Reply by mrjones2014** [reply]
Thanks, glad you like it!

The bg is handled by just having separate highlights for `Normal` and `NormalNC` (normal non current) groups. 

The window borders were coming from `colorful-winsep.nvim` but I’ve since removed it because it basically duplicates your windows as floating windows and it’s kinda buggy and jank.

**Comment by mrjones2014** [comment]
[Screencast From 2025-04-06 10-32-56.webm](https://github.com/user-attachments/assets/66afdeb7-a637-44eb-8b40-c4161db5de58)
